mpmp/src/main/Client.java

```diff
@@ -20,7 +20,6 @@
 	public Client(Socket sock) throws IOException {
 		super(sock);
 		clients.add(this);
-		send("Willkommen, Genosse! Subscriben Sie!");
```

Comment:
Yes, I'm sorry :unamused:.
However, it was simpler than adding a `Willkommen,` command.
Oh, I have an idea: simply prepend `+JAWOHL` and it will be ignored. :smile: